README.md

```diff
@@ -1 +1,180 @@
-# TDD-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+
+  <h3><b>TDD Project
+
+</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[TDD Project
+\] ](#-TDD Project
+)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [TDD Project
+] <a name="about-project"></a>
+In this project you will implement a class with some methods, but you will do it by doing test-driven development (TDD)
+## 🛠 Built With <a name="built-with"></a>
+Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a https://www.ruby-lang.org/en/>Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **TDD concept.**
+- **Unit tests using Rspec.**
+- **Implement classes and objects in Ruby.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installed and running.
+- You should be able to use IRB.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd TDD-project
+  git clone git@github.com:tamoorsaeed22/TDD-project.git
+```
+
+
+### Install
+
+Install this project with:
+```
+  cd TDD-project
+  gem install
+
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  None
+
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+```
+- Run rubocop
+```
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Tamoor Saeed**
+
+- GitHub: [@githubhandle](https://github.com/tamoorsaeed22)
+- Twitter: [@twitterhandle](https://www.linkedin.com/in/tamoor-saeed-58912a233/)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```

Comment:
![image](https://github.com/tamoorsaeed22/TDD-project/assets/93039359/e8f1861d-dca1-42cf-b57b-9af11b433703)

- [ ] It would be great if you fix your LinkedIn link because right now it is broken and giving me the an error which you can see in the image above ⬆️ 😨 Kindly fix it to make your readme descriptive and professional 👍

Reply:
@HAMMAS-SALEEM Yes I saw that was an error and I fixed that issue as the profile is opening by clicking on the LinkedIn link.